docs/quickstarts/react.mdx

```diff
@@ -16,18 +15,15 @@ description: Learn how to use Clerk to quickly and easily add secure authenticat
     {
       title: "Set up a Clerk application",
       link: "https://clerk.com/docs/quickstarts/setup-clerk"
-    },
-    {
-      title: "Create a React application using Vite",
-      link: "https://vitejs.dev/guide/#scaffolding-your-first-vite-project"
     }
   ]}
 >
+- Create a new React application using Vite
```

Comment:
I've added this here and removed it from the "beforeYouStart" section, because it's actually Step 1 of this tutorial, and not done as a prerequisite.